### PR TITLE
Update to PHPMailer 5.2.21

### PR DESCRIPTION
**Pull request by mbabker**
### Summary of Changes

Updates PHPMailer to 5.2.21.  Composer is run with 1.3.0 which explains the ClassLoader and timestamp changes.

### Testing Instructions

Mail sending works

### Documentation Changes Required

N/A

### CODE REVIEW

**Comment by yvesh** [comment]
I have tested this item :white_check_mark: successfully on 16e7dec2c830fb36ef12104009e17608cd129f52<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13388">issues.joomla.org/tracker/joomla-cms/13388</a>.</sub>

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 16e7dec2c830fb36ef12104009e17608cd129f52<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13388">issues.joomla.org/tracker/joomla-cms/13388</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13388">issues.joomla.org/tracker/joomla-cms/13388</a>.</sub>

File: composer.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
         "ircmaxell/password-compat": "1.*",
         "leafo/lessphp": "0.5.0",
         "paragonie/random_compat": "~1.0",
-        "phpmailer/phpmailer": "^5.2.18",
+        "phpmailer/phpmailer": "^5.2.20",
         "symfony/polyfill-php55": "~1.2",
```
Comment (bertmert) [inline composer.json]: 5.2.2**1** ?

File: composer.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
         "ircmaxell/password-compat": "1.*",
         "leafo/lessphp": "0.5.0",
         "paragonie/random_compat": "~1.0",
-        "phpmailer/phpmailer": "^5.2.18",
+        "phpmailer/phpmailer": "^5.2.20",
```
Comment (mbabker) [inline composer.json]: It's fine as is.  This declares that any PHPMailer version on the current major series greater than or equal to this one is fine.  It'll allow anything `>=5.2.20 && <6.0`.

**Comment by smz** [comment]
I think this is important enough to justify a quick fix release...

**Comment by mbabker** [comment]
No, it's not.  See https://developer.joomla.org/security-centre/668-20161205-phpmailer-security-advisory.html and associated references.

**Comment by Bakual** [comment]
> I think this is important enough to justify a quick fix release...

See https://developer.joomla.org/security-centre/668-20161205-phpmailer-security-advisory.html
No action required for Joomla as the vulnerability isn't useable in core.

**Comment by smz** [comment]
@mbabker & @Bakual: OK, thanks, I trust your judgement...

**Pull request merged**